skt/kerneltree.py

```diff
@@ -199,6 +199,16 @@ def get_commit_hash(self, ref=None):
         """
         return self.get_commit_details(ref, show_format='%H')
 
+    def get_commit_subject(self, ref=None):
+        """
+        Get the subject line of a commit specified by an optional ref.
+        Args:
+            ref: A reference to a commit in a git repo to be inspected.
```

Comment:
Please see if you could mention the default value, and what `None` would mean.

Reply:
Fixed.